python/lsst/ts/criopy/gui/formators.py

```diff
@@ -0,0 +1,182 @@
+    fmt : `str`, optional
+        Format string. See Python formatting function for details. Defaults to
+        'd' for decimal number.
+    unit : `astropy.units`, optional
+        Variable unit. Default is None - no unit
+    convert : `astropy.units`, optional
+        Convert values to this unit. Default is None - no unit. If provided,
+        unit must be provided as well.
+    is_warn_func : `func`, optional
+        Function evaluated on each value. If true is returned, value is assumed
+        to be in warning range and will be color coded (displayed in warning
+        text). Default is None - no color coded warning value.
+    is_err_func : `func`, optional
+        Function evaluated on each value. If true is returned, value is assumed
+        to be in warning range and will be color coded (displayed in warning
+        text). Default is None - no color coded error value.
+    """
+
+    def __init__(
+        self,
+        field: str | None = None,
+        fmt: str = "d",
+        unit: str | u.Unit | None = None,
+        convert: u.Unit | None = None,
+        is_warn_func: typing.Callable[[float], bool] | None = None,
+        is_err_func: typing.Callable[[float], bool] | None = None,
+    ):
+        self._field = field
+
+        self.fmt = fmt
+        if isinstance(unit, str):
+            unit = u.Unit(unit)
+
+        if unit is not None:
+            assert issubclass(unit.__class__, u.UnitBase)
+
+        if convert is not None:
+            if unit is None:
+                raise RuntimeError("Cannot specify conversion without input units!")
+            self.scale = unit.to(convert)
+            self.unit_name = convert.to_string()
+        elif unit is not None:
+            self.scale = 1
+            self.unit_name = unit.to_string()
+        else:
+            self.scale = 1
+            self.unit_name = ""
+
+        # we can display some units better using unicode
+        aliases = {
+            "deg_C": "°C",
+            "1 / min": "RPM",
+            "m N": "N m",
+        }
+        try:
+            self.unit_name = aliases[self.unit_name]
+        except KeyError:
+            pass
+
+        self.unit_name = re.sub(r"\bdeg[^;]", "°", self.unit_name)
+
+        # s2, s3 using sup
+        self.unit_name = self.unit_name.replace("s2", "s<sup>2</sup>")
+        self.unit_name = self.unit_name.replace("s3", "s<sup>3</sup>")
+
+        self.unit_name = " " + self.unit_name
+
+        self.unit = unit
+        self.convert = convert
+        self.is_warn_func = is_warn_func
+        self.is_err_func = is_err_func
+
+    def format(self, data: BaseMsgType) -> str:
+        """Does the formating.
+
+        Parameters
+        ----------
+        data : `BaseMsgType`
+            New data, in SALobj/ts_xml structure.
+
+        Returns
+        -------
+        text : `str`
+            String to display in the label."""
+        assert self._field is not None
```

Comment:
Returns
        -------
        text : `str`
            String to display in the label.
"""

Reply:
Changed according to Numpy doc guide to :
```
Returns
-----------
str
    String..
```